dev/devicelab/manifest.yaml

```diff
@@ -279,6 +279,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  complex_layout_scroll_perf__devtools_memory:
+    description: >
+      Measures memory usage of the scroll performance test using DevTools.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```

Comment:
Can we run this on Linux instead of mac? We have more Linux machines AFAIK.  @digiter in case I'm wrong.

Reply:
They are roughly the same, but I prefer Linux ones as they are more stable.

Reply:
Done.